site/public/mjs/sql-toolbox.js

```diff
@@ -77,6 +77,12 @@ export async function runSqlQuery() {
 
 export function init() {
     document.getElementById('run-sql-btn').addEventListener('click', () => runSqlQuery());
+    $('#sql-database-schema').on('click', () => {
+        $('#sql-database-schema-content').toggle();
+    });
+    $('.sql-database-table').on('click', (e) => {
+        $(e.target).next('.sql-database-columns').toggle();
+    });
```

Comment:
```suggestion
    document.getElementById('sql-database-schema').addEventListener('click', () => {
        document.getElementById('sql-database-schema-content').style.display =
            document.getElementById('sql-database-schema-content').style.display === 'block'
                ? 'none'
                : 'block';
    });
    document.querySelectorAll('.sql-database-table').forEach((elem) => {
        elem.addEventListener('click', (e) => {
            e.target.nextElementSibling.style.display =
                e.target.nextElementSibling.style.display === 'block'
                    ? 'none'
                    : 'block';
        });
    });
```

Easy enough to rewrite this to not require jQuery, which I'd like to avoid as trying to import it both as a regular code and module import gets messy.